### ISSUE TITLE
Urgent issue! Install Dependency not working when build app.

### ISSUE THREAD

**Issue by devaniumesh** [howljs/react-native-calendar-kit#79]
I just created **react native blank app** with below configuration, no other code or library installed.
```
node version = v18.17.0
npm version = 9.6.7
Open JDK = 11.0.16.1 2022-08-12 LTS
```

![image](https://github.com/howljs/react-native-calendar-kit/assets/10271644/f608de92-88af-4f9f-8993-d3defd75e33e)

![image](https://github.com/howljs/react-native-calendar-kit/assets/10271644/af36f37e-3b79-4442-811d-b10d62176a22)

**If remove your dependency then it works perfectly with build success.**
But when i use use your dependency it always show build failed.

Urgent help needed.

**Comment by howljs** [comment]
Have you done this step?
<img width="898" alt="Screenshot 2023-07-27 at 10 33 11" src="https://github.com/howljs/react-native-calendar-kit/assets/33460888/632296b5-e2ec-43f2-b4db-a92fb0bf0f92">

**Comment by devaniumesh** [comment]
Yes, I have already did that. You can also see my first screenshot with that command. From that dependency, only @shopify/flash-list dependency cause that problems.

**Comment by devaniumesh** [comment]
Please update your library with latest react & node version. I am still not able to use your library, getting same issue.

You are using which version of below 
node ?
java jdk ?
npm ?

**Comment by howljs** [comment]
Please try to downgrade the library to this version

     "@shopify/flash-list": "^1.3.1",

**Comment by devaniumesh** [comment]
It's showing various deprecated errors and build faild for Android.

For iOS it's working fine. It causes issue with react native with CLI in
Android

For expo app, it's working. For react native CLI it's not working in
Android. Display error as I mentioned earlier in screenshot

Approx how much time it will take to resolve this issue ? I hope it comes
within a week or so.


Thanks for your support 🙏

**Comment by howljs** [comment]
Sorry for the delay, you can try this.
https://gist.github.com/danielcshn/7aa57155d766d46c043fde015f054d40

![Screenshot 2023-07-31 at 09 16 30](https://github.com/howljs/react-native-calendar-kit/assets/33460888/d29cfd06-20c8-432c-ba0b-bf64e0f2af57)

![Screenshot 2023-07-31 at 09 16 54](https://github.com/howljs/react-native-calendar-kit/assets/33460888/d46de915-a5d6-47fb-8f4c-e3c414d1b102)

**Comment by devaniumesh** [comment]
> Please try to downgrade the library to this version. Currently, I'm rewriting a new version and won't use flash-list anymore
> 
> ```
>  "@shopify/flash-list": "^1.3.1",
> ```

Any updates regarding launch next version ?

**Comment by devaniumesh** [comment]
> Sorry for the delay, you can try this. https://gist.github.com/danielcshn/7aa57155d766d46c043fde015f054d40
> 
> ![Screenshot 2023-07-31 at 09 16 30](https://user-images.githubusercontent.com/33460888/257095120-d29cfd06-20c8-432c-ba0b-bf64e0f2af57.png)
> 
> ![Screenshot 2023-07-31 at 09 16 54](https://user-images.githubusercontent.com/33460888/257095168-d46de915-a5d6-47fb-8f4c-e3c414d1b102.png)

Great fix, just for temporary solution.

But it also show deprecation error for shopify-list in terminal.

**Comment by howljs** [comment]
Fixed in v2.0.0
